include/crpropa/Source.h

```diff
@@ -319,16 +319,18 @@ class SourceUniformBox: public SourceFeature {
 
 /**
  @class SourceUniformCylinder
- @brief Uniform distribution of source positions inside the volume of a cylinder
+ @brief Uniform distribution of source positions inside the volume of a cylinder. 
```

Comment:
Would be great to add explicit info about the orientation of the cylinder's axis.